test/lit.cfg

```diff
@@ -902,19 +925,18 @@ elif run_os == 'linux-androideabi':
     subst_target_swift_ide_test_mock_sdk = config.target_swift_ide_test
     subst_target_swift_ide_test_mock_sdk_after = ""
     config.target_swiftc_driver = (
-        "%s -target %s -sdk %r %s %s %s" %
+        "%s -target %s -sdk %r -tools-directory %s %s %s %s -use-ld=%s" %
         (config.swiftc, config.variant_triple, config.variant_sdk,
-         android_linker_opt, resource_dir_opt, mcp_opt))
+         tools_directory, android_linker_opt, resource_dir_opt, mcp_opt,
+         config.android_linker_name))
     config.target_swift_modulewrap = (
         '%s -modulewrap -target %s' %
         (config.swiftc, config.variant_triple))
     config.target_clang = (
-        "clang++ -target %s %s" %
-        (config.variant_triple, clang_mcp_opt))
+        "clang++ -target %s %si %s" %
```

Comment:
Why is there an extra `i` after the `%s`?

Reply:
Stupid vim.